packages/clerk-js/src/ui/elements/Action/ActionOpen.tsx

```diff
@@ -1,16 +1,29 @@
 import type { PropsWithChildren } from 'react';
+import React, { useEffect, useRef } from 'react';
 
 import { useActionContext } from './ActionRoot';
 
 type ActionOpenProps = PropsWithChildren<{ value: string }>;
```

Comment:
Let's move this below the `<ScrollWrapper />` as it relates to  `<Action.Open />`

Reply:
Done!